spring-boot/admin/src/main/resources/templates/admin/fragments/scripts.html

```diff
@@ -92,7 +91,8 @@
             <script>
                 $( document ).ready(function() {
                     $('table[class*="default-data-table"]').DataTable( {
-                            "scrollX": true
+                        "sScrollX": "100%",
+                        "sScrollXInner": "110%",
```

Comment:
This fixes the bug.

However, it is not very user-friendly, as the table contents _never_ fit to the page width, horizontal scrolling is _always_ required, even if there is hardly any data in the table (I assume this is because of the `110%` setting, which means "use more space than available").

Could these parameters be changed to take only 100% by default?

Reply:
Yes, I changed the sScrollXInner to 100% and now it is working well